src/main/scala/wdlTools/typing/package.scala

```diff
@@ -1,11 +1,13 @@
 package wdlTools.typing
 
+import java.net.URL
+
 import wdlTools.syntax.TextSource
 
 // Type error exception
 final class TypeException(message: String) extends Exception(message) {
-  def this(msg: String, text: TextSource) =
-    this(s"${msg} at ${text}")
+  def this(msg: String, text: TextSource, docSourceURL: Option[URL] = None) =
+    this(s"${msg} at ${text}${docSourceURL.map(url => s" in ${url.toString}").getOrElse("")}")
```

Comment:
Can you please calculate the document string in a separate line?

Reply:
Same as above